src/test/java/com/internship/frejaeidjmeterplugin/jmeter/frejaRequests/AuthenticationServiceTest.java

```diff
@@ -0,0 +1,84 @@
+package com.internship.frejaeidjmeterplugin.jmeter.frejaRequests;
+
+import com.verisec.frejaeid.client.enums.MinRegistrationLevel;
+import com.verisec.frejaeid.client.exceptions.FrejaEidClientInternalException;
+import com.verisec.frejaeid.client.exceptions.FrejaEidClientPollingException;
+import com.verisec.frejaeid.client.exceptions.FrejaEidException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class AuthenticationServiceTest {
```

Comment:
add get results tests

Reply:
I tested get result in first test method, i can't test that without reference, and if i put reference as atribut of class i get null pointer.  I can't send init again in get result as well because of security policy.